### PR TITLE
WF-62406-Add UG Documentation to Modify Footnote and Endnote content

### PR DESCRIPTION
**Pull request by sharonaumao**
Added UG Documentation to Footnotes-and-Endnotes md fileformat for Modify Footnote and Endnote content.

### CODE REVIEW

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 1]
```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:704]: Should have sentence about the topic

You can modify footnote and endnote contents in an existing Word document.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 2]
```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
...
+The following code example shows how to modify the footnote/endnote content from the word document.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:705]: The following code example shows how to modify the footnote and endnote content from existing word document.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 3]
```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
...
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:709]: Every XML comment should end with period.

Also, Verb should not be Verb+s.  use, the normal verb

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 4]
```diff
@@ -700,6 +700,205 @@ using (WordDocument document = new WordDocument())
...
+//Loads the template document
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:710]: //Load an existing Word document.
Using (WordDocument document = new WordDocument("Template.docx")😉
{ //Access the footnote from Word document.
WFootnote footnote = paragraph.ChildEntities[0] as WFootnote;
//Clear the footnote contents.
footnote.TextBody.ChildEntities.Clear();
//Add new paragraph to footnote
WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
//Set the footnote character format.
footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
//Append footnote text.
footnoteParagraph.AppendText(" Footnote is modified.");
//Access endnote in Word document.
WFootnote endnote = paragraph.ChildEntities[0] as WFootnote;
//Clear endnote content.
endnote.TextBody.ChildEntities.Clear();
//Add new paragraph to endnote
WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
//Sets the endnote character format.
endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
//Append endnote text.
endnoteParagraph.AppendText(" Endnote is modified.");
//Save the Word document.
document.Save("Sample.docx", FormatType.Docx);
}

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 1 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:705]: I have added.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 3 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:710]: I have done so

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 5]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+//Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:903]: @sharonaumao - Please add suitable sample URL link.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 6]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+//Access paragraph in Word document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
+//Get the footnote at index 0.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:716]: //Access footnote in paragraph.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 7]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+footnote.TextBody.ChildEntities.Clear();
+//Add new paragraph to body of footnote.
+WParagraph footnoteParagraph = footnote.TextBody.AddParagraph() as WParagraph;
+//Set the footnote character format.
+footnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append footnote text.
+footnoteParagraph.AppendText(" Footnote is modified.");
+//Access paragraph in Word document.
+paragraph = document.Sections[2].Paragraphs[1] as WParagraph;
+//Get the endnote at index 0.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:728]: //Access endnote in paragraph.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 6 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: @MohanaselvamJothi - I have corrected.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 7 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: @MohanaselvamJothi - I have made correction including all the remaining platforms.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 8]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+The following code example shows how to modify the footnote and endnote content from an existing word document.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+//Access paragraph in a Word document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:715]: @sharonaumao - Please align the code snippets properly as similar to below UG topic,
https://github.com/syncfusion-content/fileformat-docs/edit/master/File-Formats/DocIO/mail-merge/mail-merge-options.md

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 9]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+Save(stream, "Sample.docx");
+//Please refer to the below link to save the Word document in UWP platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+                }
+            }
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as Stream.
+using (FileStream inputStream = new FileStream(@"../../../Input.docx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:819]: @sharonaumao - Please remove the unwanted hint path from here.

Comment (Ramaraj-Marimuthu) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:819]: @sharonaumao - Please modify the FileShare type.

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md [hunk 10]
```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
...
+endnote.TextBody.ChildEntities.Clear();
+//Add a new paragraph to the body of the endnote.
+WParagraph endnoteParagraph = endnote.TextBody.AddParagraph() as WParagraph;
+//Set the endnote character format.
+endnote.MarkerCharacterFormat.SubSuperScript = SubSuperScript.SuperScript;
+//Append the endnote text.
+endnoteParagraph.AppendText(" Endnote is modified.");
+//Save the Word document to the MemoryStream.
+MemoryStream stream = new MemoryStream();
+document.Save(outputStream, FormatType.Docx);
```
Comment (Ramaraj-Marimuthu) [inline File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md:891]: @sharonaumao - Please change the `MemoryStream` object naming.

**Review by Ramaraj-Marimuthu — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 8 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: @Ramaraj-Marimuthu -Done

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 9 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: @Ramaraj-Marimuthu -Done

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 10 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: @Ramaraj-Marimuthu -Done

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 4 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: Done

File: File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md (hunk 2 above)
Comment (sharonaumao) [reply File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md]: Added additional information as required

**Review by Ramaraj-Marimuthu — APPROVED** [review]
I don't have concerns in this content.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**